components/gitpod-protocol/src/permission.ts

```diff
@@ -6,18 +6,13 @@
 
 // see below for explanation
 export const Permissions = {
-    monitor: undefined,
     developer: undefined,
-    enforcement: undefined,
```

Comment:
This was used to guard the `/api/enforcement/*` endpoints, which no longer exist.